https://issues.redhat.com/browse/ACM-13654 (#6842)
open-cluster-management/rhacm-docs: observability/observe_environments_intro.adoc

[#observing-environments-intro]
= Observability service

Observability can help you identify and assess performance problems without additional tests and support. The {acm} observability component is a service you can use to understand the health and utilization of clusters, and workloads across your fleet. By using the observability service, you are able to automate and manage the components that are within observability. 

> By default, multicluster observability operator is enabled during the installation of {acm-short}. The `observability-endpoint-operator` is automatically deployed to each imported or created managed cluster. This controller starts a metrics collector that collects the data from {ocp} Prometheus, then sends the data to the {acm-short} hub cluster.

Author: ```suggestion
Observability service uses existing and widely-adopted observability tools from the open source community. By default, multicluster observability operator is enabled during the installation of {acm-short}. Thanos is deployed within the hub cluster for long-term metrics storage. The `observability-endpoint-operator` is automatically deployed to each imported or created managed cluster. This controller starts a metrics collector that collects the data from {ocp} Prometheus, then sends the data to the {acm-short} hub cluster.
```

> //== Observability open source components
> 
> //What other open source components are being used? The only thing mentioned was Thanos. Is Alertmanager and Prometheus also open source? 

Author: ```suggestion
```

[#observing-environments-intro]
= Observability service

Observability can help you identify and assess performance problems without additional tests and support. The {acm} observability component is a service you can use to understand the health and utilization of clusters, and workloads across your fleet.

By using the observability service, you are able to automate and manage the components that are within observability. By default, multicluster observability operator is enabled during the installation of {acm-short}.

> When you enable the observability service, the `observability-endpoint-operator` is automatically deployed to each imported or created managed cluster. This controller starts a metrics collector that collects the data from {ocp} Prometheus, then sends the data to the {acm-short} hub cluster.

Author: ```suggestion
Observability can help you identify and assess performance problems without additional tests and support. The {acm} observability component is a service you can use to understand the health and utilization of clusters, and workloads across your fleet. By using the observability service, you are able to automate and manage the components that are within observability. 

By default, multicluster observability operator is enabled during the installation of {acm-short}. The `observability-endpoint-operator` is automatically deployed to each imported or created managed cluster. This controller starts a metrics collector that collects the data from {ocp} Prometheus, then sends the data to the {acm-short} hub cluster.
```

[#observing-environments-intro]
= Observability service

> The observability component is a service you can use to understand the health and utilization of clusters across your fleet. By default, multicluster observability operator is enabled during the installation of {acm-short}. 

Reviewer: Why you need Observability or why observability is important?

> //Why is Observability important on a multi-cluster level?

Author: ```suggestion
Observability can help you identify and assess performance problems without additional testing. 
```

> The observability component is a service you can use to understand the health and utilization of clusters and your workloads across your fleet. By default, multicluster observability operator is enabled during the installation of {acm-short}.

Reviewer: Addition - When you are managing multiple clusters, automating and managing Observability for those clusters is hard. With ACM observability service, we leverage the powers of ACM to give you an automatic monitoring setup experience for all the clusters you manage.

> The observability component is a service you can use to understand the health and utilization of clusters and your workloads across your fleet. By using the observability service, you are able to automate and manage the components that are within observability. By default, multicluster observability operator is enabled during the installation of {acm-short}.

Author: ```suggestion
Observability can help you identify and assess performance problems without additional tests and support. The {acm} observability component is a service you can use to understand the health and utilization of clusters, and workloads across your fleet.

By using the observability service, you are able to automate and manage the components that are within observability. By default, multicluster observability operator is enabled during the installation of {acm-short}.
```

//ACM Observability and what it brings to the table (big highlight on cost savings here)

> // I need help with understanding the cost savings, how does observability help the user save costs?

Reviewer: When I mention cost savings, I mean,
- Not having to maintain dedicated complex Observability tooling on your own
- Not having to have SRE-personas work and be on-call to maintain that system and instead use RH support
- Not having to pay costly Observability vendors (check with PM)

Author: Hoping that my changes on lines 4-6 answered these questions. Resolving this convo for now

> // I need help with understanding the cost savings, how does observability help the user save costs?

Author: ```suggestion
```

When you enable the observability service,  the `observability-endpoint-operator` is automatically deployed to each imported or created managed cluster. This controller starts a metrics collector that collects the data from {ocp} Prometheus, then sends the data to the {acm-short} hub cluster.

//== Observability open source components

> //What other open source components are being used? The only thing mentioned was Thanos. Is Alertmanager and Prometheus also open source? 

Reviewer: Yes! Thanos, Prometheus, Prometheus-Operator, Alertmanager are being used, and all of these are open source.

We could mention something like "All of your Observability is powered by open source. We use existing and widely-adopted observability tooling from the upstream communities, to provide you with an great Observability service.

- Thanos
- Prometheus
- Alertmanager

And link to them with maybe a line each about why we use them?"

Author: Would you be opposed to describing the open source components in the arch file?

Reviewer: Not opposed, we would need to include them in arch file anyway, just wanted to have it in intro
